ConveyorSkills/.gitignore

```diff
@@ -0,0 +1,4 @@
+/.vscode
```

Comment:
Would one .gitignore in the root be better than having multiple project .gitignores with the same ignored files.